supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -10,3 +10,5 @@
 
 [[long-code-examples]]
 == Long code examples in procedures
+
+All code blocks, regardless of length, should be necessary, accurate, and helpful. Code blocks should be as copy-and-paste friendly as possible, with the exception of variables and callouts only. The length of the block is irrelevant, within reason, if the code block follows these guidelines. For more information about code sample suggestions, see IBM Style Guide pp. 198-200.
```

Comment:
I'd remove the last sentence about the IBM style guide. You could really say that to any of these guidelines, plus the page numbers might not be accurate (I have an online version that has no page numbers).

Reply:
Removed